cli/running/lua/launcher.lua

```diff
@@ -20,6 +21,32 @@ local function start_instance()
         __defer_update = true
     }
 
+    -- Make stdout linebuffered. gh-36.
+    -- By default (when using "glibc") "stdout" is line buffered when connected
+    -- to a TTY and block buffered (one page 4KB) when connected to a pipe / file.
+    -- This is not how we want to log work, so set "stdout" to line buffered mode
```

Comment:
It is not about log, but about `print()` in Lua, see https://github.com/tarantool/tt/issues/36#issuecomment-1106915036.

Reply:
Comment rewritten.